From b374356a35247a21484ca37bdce2a87c34905e7c Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Wed, 22 Jun 2011 17:56:55 +0800
Subject: [PATCH] ENGR00151900 vpu: fix build failure for mx3 platform

Fix build failure for mx3 platform due to ENGR00151494 push.

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/mach-mx3/system.c |   11 ++++++++++-
 1 files changed, 10 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx3/system.c b/arch/arm/mach-mx3/system.c
index a2f5442..8cfeae3 100644
--- a/arch/arm/mach-mx3/system.c
+++ b/arch/arm/mach-mx3/system.c
@@ -1,7 +1,7 @@
 /*
  * Copyright (C) 1999 ARM Limited
  * Copyright (C) 2000 Deep Blue Solutions Ltd
- * Copyright 2004-2010 Freescale Semiconductor, Inc.
+ * Copyright 2004-2011 Freescale Semiconductor, Inc.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -91,3 +91,12 @@ void arch_idle(void)
 	}
 }
 
+void mxc_pg_enable(struct platform_device *pdev)
+{
+}
+EXPORT_SYMBOL(mxc_pg_enable);
+
+void mxc_pg_disable(struct platform_device *pdev)
+{
+}
+EXPORT_SYMBOL(mxc_pg_enable);
-- 
1.5.4.4

